setup.py

```diff
@@ -19,6 +19,7 @@
     "treeherder-client",
     "PGPy",
     "buildtools",
+    "python-jose",
```

Comment:
should we put the other python modules in here that you added to the puppet env?